client/src/components/Email/EmailTemp.js

```diff
@@ -0,0 +1,180 @@
+import React from "react";
+import {Container, Dropdown, Form, Grid, Header} from "semantic-ui-react";
+
+const templateOptions = [
+    {
+        key: "SW",
+        text: "Social Worker",
+        value: "1",
+    },
+    {
+        key: "Doc",
+        text: "Doctor",
+        value: "2",
+    }
+];
+
+
+class EmailTemp extends React.Component {
```

Comment:
this is no longer temporary, we should rename this

Reply:
Temporary? I thought temp stood for template?

Reply:
oh, well renaming it to EmailTemplate might be less ambiguous but you can leave it if you want